docs/patched-deps.txt

```diff
@@ -0,0 +1,2 @@
+# fix from here: https://github.com/executablebooks/MyST-NB/pull/597
```

Comment:
Why is this necessary on top of the pyproject.toml?

Reply:
because
- you can’t directly specify deps in .readthedocs.yml, only through referencing paths
- you can’t specify direct dependencies in standard metadata, only tool-specific metadata (e.g. if we’d use hatch or poetry to create our envs, we could use that)

Reply:
It certainly works to install the package + extra dependencies listed in pyproject.toml
```
python:
  install:
    - method: pip
      path: .
      extra_requirements:
        - docs
```

Not that its super important if it's solved one way or another, was mostly curious.